### PR TITLE
Snom linekeys support

### PR DESCRIPTION
**Pull request by alepolidori**
All the linekeys with X into the first column of the following document has been imlpemented:

image

Expected input are the "value" column and the output for snom phones are in the last one.

### CODE REVIEW

File: public/provisioning.php [hunk 1]
```diff
@@ -149,8 +149,11 @@
     // Add provisioning_complete variable
     $scope_data['provisioning_complete'] = '';
 
-    // Add provisioning_url_path variable
+    // Add provisioning_url_path and provisioning_url_host variables
     $scope_data['provisioning_url_path'] = $config['provisioning_url_path'];
+    if(empty($scope_data['provisioning_url_host'])) {
+        $scope_data['provisioning_url_host'] = gethostname();
```
Comment (DavidePrincipi) [inline public/provisioning.php]: Can you split this change to a distinct PR?

File: public/provisioning.php [hunk 2]
```diff
@@ -169,9 +172,9 @@
         // Load twig template
         $loader = new \Twig\Loader\FilesystemLoader($config['ro_dir'] . 'templates/');
         $twig = new \Twig\Environment($loader,['autoescape' => false]);
-	$response = $response->getBody()->write($twig->render($template,$scope_data));
-	$this->logger->debug($request->getMethod() ." " . $request->getUri() .' Result: 200 ok ' . __FILE__.':'.__LINE__);
-	return $response;
+        $response = $response->getBody()->write($twig->render($template,$scope_data));
```
Comment (DavidePrincipi) [inline public/provisioning.php]: Avoid whitespace changes

File: public/provisioning.php (hunk 1 above)
Comment (alepolidori) [reply public/provisioning.php]: Done: https://github.com/nethesis/tancredi/pull/48

File: public/provisioning.php (hunk 2 above)
Comment (alepolidori) [reply public/provisioning.php]: Done

**Pull request merged**